src/customTheme.ts

```diff
@@ -39,11 +45,80 @@ const blueColor: ColorHues = {
   900: '#00131b',
 }
 
+// ***** New colorscheme *****
+
+// The coolGray palette contains all the shades of gray we'll use in the app.
+// Example uses: body text, headings, changelog text, page background.
+// https://tailwindcss.com/docs/customizing-colors#color-palette-reference
+
+const tailwind_coolGray: ColorHues = {
+  50: '#F9FAFB',
+  100: '#F3F4F6',
+  200: '#E5E7EB',
+  300: '#D1D5DB',
+  400: '#9CA3AF',
+  500: '#6B7280',
+  600: '#4B5563',
+  700: '#374151',
+  800: '#1F2937',
+  900: '#111827',
+}
+
+// The Fuchsia palette is our primary accent.
+// Example uses: call to action button, primary action button.
+// https://tailwindcss.com/docs/customizing-colors#color-palette-reference
+
+const tailwind_Fuchsia: ColorHues = {
+  50: '#FDF4FF',
+  100: '#FAE8FF',
+  200: '#F5D0FE',
+  300: '#F0ABFC',
+  400: '#E879F9',
+  500: '#D946EF',
+  600: '#C026D3',
+  700: '#A21CAF',
+  800: '#86198F',
+  900: '#701A75',
+}
+
+// The Sky palette is our secondary accent.
+// Example uses: secondary button, version badge.
+// https://tailwindcss.com/docs/customizing-colors#color-palette-reference
+
+const tailwind_Sky: ColorHues = {
+  50: '#F0F9FF',
+  100: '#E0F2FE',
+  200: '#BAE6FD',
+  300: '#7DD3FC',
+  400: '#38BDF8',
+  500: '#0EA5E9',
+  600: '#0284C7',
+  700: '#0369A1',
+  800: '#075985',
+  900: '#0C4A6E',
+}
+
+// These are our normal text colors.
+// Do NOT use when you have text that goes on a colored background.
+// Text on colored backgrounds should use a suitable hue from the background color scheme.
+
+const primaryText = '#111827'
+const secondaryText = '#4B5563'
+const tertiaryText = '#6B7280'
```

Comment:
I just realized I need to figure out 3 good shades of gray for use on our darkmode as well.
That way all possible text colors are nicely systematized. 😄 

```js
// 3 text colors for use with light mode

const primaryTextLightmode
const secondaryTextLightmode
const tertiaryTextLightmode

// 3 text colors for use with dark mode

const primaryTextDarkmode
const secondaryTextDarkmode
const tertiaryTextDarkmode
```

Reply:
The text colors will al be a shade of coolGray (I think). Is there a way to just link to the correct `coolGray` shade? I'm looking to do something like this:

```typescript
const primaryText = tailwindCoolgray.900
```

Reply:
You definitely can assign it like that, just changing the syntax a bit: `const primaryText = coolGray['900']`

I'm not sure how text colors are assigned yet, so I need some time to investigate that. Anyway, just focus on finding the proper colors so we can use them manually in some components, and then I can set it as the corresponding default token.